training/distributed_training/pytorch/model_parallel/gpt-j/train_gptj_smp_notebook.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>Spell out "SMP" in the first sentence, i.e.:</p><p> </p><p>This notebook walks you through how to train the Eleuther GPT-J model with SageMaker's model parallelism (SMP).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>